### ISSUE TITLE
Not able to Configure Azure Activity logs

### ISSUE THREAD

**Issue by abhimanmohan** [MicrosoftLearning/AZ500-AzureSecurityTechnologies#156]
# Module: 04
#### Step: 03

I am not getting the option of Configure Azure Activity logs, tried every way out not able to do, need assistance 
Repro steps:

1.
1.
1.

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/129853030-1a213417-3e6a-4747-b2c7-4dc33bf71913.png)

**Comment by thegreatmayank** [comment]
Yes the procedure is changed now. you may find the new procedure over here https://docs.microsoft.com/en-us/azure/sentinel/connect-azure-activity#set-up-the-azure-activity-log-connector

**Comment by abhimanmohan** [comment]
after doing the steps mentioned on the link https://docs.microsoft.com/en-us/azure/sentinel/connect-azure-activity#set-up-the-azure-activity-log-connector  , still its showing same page 
![image](https://user-images.githubusercontent.com/64397100/129857321-48cca401-746d-442a-a136-baee7377dba9.png)

**Comment by thegreatmayank** [comment]
These steps create a policy, ensure that the policy is 100 percent compliant, once it is so you will see this as connected. It does take some time and you may have to manually start the remediation task for the policy if it is not executed automatically.

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/129859233-fe00e502-a86b-4336-ad60-b321dcadb9d4.png)

policy is complient

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/129859461-4e310ba2-e93c-4178-8349-4d0413152ea9.png)

But still its showing like this, so can I move to the next step or have to wait ?

**Comment by thegreatmayank** [comment]
give it some time policy has to be replicated globally. You will see the connector as connected soon, just go back to the connector tab (where you chose azure activity) and refresh

**Comment by thegreatmayank** [comment]
till then you may proceed with the task of creating logic app it must not have any dependency on this.

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/129860166-edd46120-4b3b-47d1-aa81-3da87f9c0bc1.png)

:(

**Comment by thegreatmayank** [comment]
leave it for some time and proceed to next task after completing that come back here and proceed with other steps of this task

**Comment by thegreatmayank** [comment]
I hope you were able to complete the lab?

**Comment by abhimanmohan** [comment]
No I am still stuck at the same place its not showing till yet : 

![image](https://user-images.githubusercontent.com/64397100/130237579-c9c5a44a-835f-415a-8770-facb9f845eea.png)

I left that part and moved forward :
in Task 6
Step 7 and 8 is not showing the result , even if I followed all the previous steps:

![image](https://user-images.githubusercontent.com/64397100/130237845-3a0dd73f-ece8-49f2-836e-4afd4fcca34d.png)

I don't know what to do I am not able to complete this Lab because of this issues

**Comment by thegreatmayank** [comment]
> ![image](https://user-images.githubusercontent.com/64397100/129859233-fe00e502-a86b-4336-ad60-b321dcadb9d4.png)
> 
> policy is complient

As I was reviewing it there is no resource compliant in it maybe the scope of the policy is not set properly.

**Comment by abhimanmohan** [comment]
I followed this step:
![image](https://user-images.githubusercontent.com/64397100/130267126-d95bf6a7-d686-49b7-a888-0be8a7752eba.png)


one request can you please try this Lab one from Task 1 to 6, I think even you will get the same issue, I saw some videos on youtube everyone was getting the issue

**Comment by thegreatmayank** [comment]
I will take step by step screenshot of the steps as being an MCT I have given this training on these labs a couple of times with the revised steps, it did take some time but eventually, it gets completed.

**Comment by thegreatmayank** [comment]
I followed the steps again and was able to connect 
![image](https://user-images.githubusercontent.com/71171403/130278385-ab45e5cd-983e-48d8-a8f2-1a1d1ce76c9c.png)
Ensure that while creating the policy you are chosing the appropriate scope and have some compliant resources in the policy like  here
![image](https://user-images.githubusercontent.com/71171403/130278483-47140590-4b0b-40e2-aab5-76d72a91f542.png)
Ensure that the scope is the subscription only and ensure to create the remediation task too. It does take time I applied the policy at 11:37 PM and got the compliance and connector as connected at 11:57PM

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/130304482-3f62aab4-01b3-4475-bb2c-3a81c630837b.png)

the location of my sentinel is East US and the Log Analytics is East US 2 . Can this be  a reason that my connector is not connecting ?

should I try with the same region of both , or reason does not matter ?

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/130304721-c4b69532-8823-4c0b-9268-9e08825abc3d.png)

its 0 for me but i followed these steps only 

![image](https://user-images.githubusercontent.com/64397100/130304761-46c0e0d9-b919-4cda-8b62-713194af59b5.png)

how to do,  this is the poly process eirght or I am missing something ?

**Comment by thegreatmayank** [comment]
I don't think the location is a matter of concern, what I think the scope is not set as a subscription for the policy, you may start lab 15 afresh and delete other resources beforehand also you may deploy a new log analytics workspace in the east us region only. you may also try revisiting the steps of lab 13 because the log analytics workspace was supposed to be created in east us under the lab instructions.

**Comment by abhimanmohan** [comment]
let me try again to delete the RG and redo it. Hope this time it will go smooth do I need to give any permissions? I have global admin permission right now .

**Comment by abhimanmohan** [comment]
@thegreatmayank  still i am getting this : 

![image](https://user-images.githubusercontent.com/64397100/130403118-60fc2d0c-d44f-486d-bfea-a665c3e40623.png)


This is my resource group till this step:

![image](https://user-images.githubusercontent.com/64397100/130403219-daad2324-0883-4b5b-97d3-efcd02d9ca92.png)

**Comment by abhimanmohan** [comment]
what to do i guess i should leave this lab! @thegreatmayank 
![image](https://user-images.githubusercontent.com/64397100/130406175-c4faea6a-1e3f-4e5a-b6d6-cf6911fce65c.png)


I am following all the steps and not getting result. :(

**Comment by abhimanmohan** [comment]
@thegreatmayank I am using free account is that a reason that my policy is not compliant?

**Comment by thegreatmayank** [comment]
> @thegreatmayank still i am getting this :
> 
> ![image](https://user-images.githubusercontent.com/64397100/130403118-60fc2d0c-d44f-486d-bfea-a665c3e40623.png)
> 
> This is my resource group till this step:
> 
> ![image](https://user-images.githubusercontent.com/64397100/130403219-daad2324-0883-4b5b-97d3-efcd02d9ca92.png)

I think tou have applied the policy on resource group... Apply it onto the subscription only... Also I have not tried it on free account I will try it tomorrow on that too and will let you know... Meanwhile... Don't lose hope..

**Comment by abhimanmohan** [comment]
sure please from free account, i will keep trying. thanks

**Comment by abhimanmohan** [comment]
![image](https://user-images.githubusercontent.com/64397100/130496109-2530428e-971b-41ad-bdf0-7d3939bbb183.png)

I have applied policy on subscription only, as you can see in the figure

**Comment by thegreatmayank** [comment]
> ![image](https://user-images.githubusercontent.com/64397100/130496109-2530428e-971b-41ad-bdf0-7d3939bbb183.png)
> 
> I have applied policy on subscription only, as you can see in the figure

Hi,
I tried doing this through a free account too and was able to do it. The scope where you have assigned this policy is to the resource group as it shows Azure Pass.../AZ500LAB131415 if it would be on the subscription level then it would have not shown the name of the resource group. like in my case as below
![image](https://user-images.githubusercontent.com/71171403/130658982-9a8988bb-16e0-4a7d-9a9e-05a139fb2407.png)
and you can see I have been able to connect to the data connector in this subscription too..
![image](https://user-images.githubusercontent.com/71171403/130659168-3087bbbb-decb-437c-ac69-be2d0f46bf5d.png)

**Comment by abhimanmohan** [comment]
Thanks @thegreatmayank I am able to do this step now, I have another query now in last step rasing another ticket for it please have a look
